scripts/release.sh

```diff
@@ -32,15 +32,23 @@ fi
 if [[ $WORKFLOW_NAME == amr ]]; then
     TAG="amr-v1.3.0"
```

Comment:
Unrelated to this PR, but is amr just locked down 1.3.0 entirely at the moment? What's going on here?

Reply:
Hey Vincent, this ugly hack was me. This release script doesn't distinguish between a beta and a full release tag, so something like `amr-v1.3.0-beta.1-0xasdf` looks exactly the same as `amr-v1.3.0` to it, which is an issue when I want to release a beta version of the workflow so I can test it in sandbox or staging, because if I do a branch release of `1.3.0-blahblah` then when I merge the changes into main the script bumps up the patch/minor/version number and tags the full release as `1.3.1`.

I want to update the script but haven't gotten around to it.

Reply:
Got it, thanks!